tests/test_sshfsmount.cpp

```diff
@@ -21,14 +21,22 @@
 #include <multipass/exceptions/sshfs_missing_error.h>
 #include <multipass/ssh/ssh_session.h>
 #include <multipass/sshfs_mount/sshfs_mount.h>
+#include <multipass/optional.h>
+#include <multipass/utils.h>
 
 #include <gmock/gmock.h>
+#include <unordered_set>
+#include <vector>
+#include <algorithm>
+#include <iterator>
 
 namespace mp = multipass;
 namespace mpt = multipass::test;
 
 using namespace testing;
 
+typedef std::vector<std::pair<std::string, std::string>> command_vector;
```

Comment:
Use CamelCase for this so it's clearer it's a type?

Reply:
Right, fixed.